templates/configurations/typeahead.interface.hbs

```diff
@@ -0,0 +1,82 @@
+<div class="form-group">
+  <label class="control-label">Select type of suggested values</label>
+  <div>
+    <div class="radio radio-icon">
+      <input type="radio" id="manual-option" name="manual-option" value="manual" v-model="optionsType" v-on:click="removeDataProvider">
+      <label for="manual-option">
+        <span class="check"><i class="fa fa-circle"></i></span> <strong>List of values</strong>
+      </label>
+    </div>
+    <div class="radio radio-icon">
+      <input type="radio" id="data-source-option" name="data-source-option" value="dataSource" v-model="optionsType" v-on:click="initDataProvider">
+      <label for="data-source-option">
+        <span class="check"><i class="fa fa-circle"></i></span> <strong>From a data source</strong>
+      </label>
+    </div>
+  </div>
+</div>
+
+<div class="form-group clearfix">
+  <div class="checkbox checkbox-icon">
+    <input type="checkbox" name="free-input" v-model="freeInput" id="free-input">
+    <label for="free-input">
+      <span class="check"><i class="fa fa-check"></i></span>
+      <span class="hide-field">Allow user to add their options</span>
+    </label>
+  </div>
+</div>
+
+<div v-show="optionsType === 'dataSource'" class="form-group clearfix">
+  <div id="data-provider"></div>
+  <label>Data field name</label>
+  <select class="form-control" :value="column" v-model="column" @change="_getDataSourceColumnValues">
+    <option value="">--- Select a data field</option>
+    <option v-for="(opt, index) in columnOptions" :value="opt">\{{opt}}</option>
+  </select>
+</div>
+<div v-if="optionsType === 'manual'" class="form-group clearfix">
+  <label class="control-label">Value options (one per line)</label>
+  <textarea v-on:input="_setOptions($event.target.value, 'options')" class="form-control">\{{ _options }}</textarea>
+</div>
+
+<div class="form-group">
+  <label>Number of list items user can select or add</label>
+  <input class="form-control" type="text" v-model.number="maxItems" />
+  <div class="secondary-label">Leave it blank to allow any amount of items or enter a number</div>
+</div>
+
+<div class="form-group">
+  <hr/>
+</div>
+
+<div class="form-group">
+  <label>Default value type</label>
```

Comment:
I see all of this being replicated from the `form.interface.hbs` file. Can you set up another .hbs file for this to avoid replicating the HTML markup? It's quite a complex set of markup to be repeated in different files.